### PR TITLE
Pom omero client 5.2.1

### PR DESCRIPTION
**Pull request by sbesson**
Follow-up of https://github.com/ome/pom-omero-client/pull/7, this PR consumes the 5.2.1 `pom-omero-client` and inherits the `formats-bsd/formats-gpl` version numbers.

### CODE REVIEW

**Comment by sbesson** [comment]
See https://ci.openmicroscopy.org/job/OMERO-DEV-merge-maven-client/30/

File: pom.xml [hunk 1]
```diff
@@ -59,6 +52,10 @@
           <id>ome.maven</id>
           <url>http://artifacts.openmicroscopy.org/artifactory/maven/</url>
         </repository>
+        <repository>
```
Comment (joshmoore) [inline pom.xml]: Shouldn't this be a temporary block in a separate PR which can be used to push the pom from staging to release?

Comment (sbesson) [inline pom.xml]: But this line is mandatory for testing the PR via Travis. Once the pom is pushed to release, happy to revert it.

Comment (joshmoore) [inline pom.xml]: Hmmm..... then when others run into the same problem? `scc merge` & `depends-on`?

Comment (sbesson) [inline pom.xml]: Tempted to propose `profile` but I need to think of the various implications.

Comment (joshmoore) [inline pom.xml]: Travis matrix testing all profiles?

**Comment by jburel** [comment]
```build.gradle``` needs to be updated too in that case
e.g. remove  ``` compile(group: 'com.google.guava', name: 'guava', version: '17.0')```, remove usage of snapshot

**Comment by jburel** [comment]
The staging repo ```http://artifacts.openmicroscopy.org/artifactory/simple/ome.staging/```
also needs to be added to gradle file.

**Comment by sbesson** [comment]
@jburel @joshmoore: I am stuck in between two thoughts here. Either we permanently add `ome.staging`  to the list of the repositories everywhere. This has the advantage of allowing us to test both staging `pom-omero-client` but also staging releases via `minimal-omero-client` and this could be part of the release process. Or we create a dedicated workflow for testing staging Maven artifacts as @joshmoore suggested in https://github.com/ome/minimal-omero-client/pull/11#discussion_r45608415.

**Comment by jburel** [comment]
Note that if we had multiple repositories, the order is important otherwise the wrong jar could be retrieved.

**Comment by sbesson** [comment]
Last commit adds the staging artifactory to the Gradle build. Staging is always resolved last in both cases (to test pre-release POMs)

**Comment by sbesson** [comment]
Last commit should test the published OMERO 5.2.1 artifacts and `pom-omero-client`

**Comment by jburel** [comment]
version in ``gradle`` file should be changed ``version '5.2.1-SNAPSHOT'```

**Comment by sbesson** [comment]
Pushed

**Comment by jburel** [comment]
Thanks

**Comment by jburel** [comment]
https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-maven-client/50/ is green
Merging
